deployment/terraform/assets/cluster.tf

```diff
@@ -85,6 +85,7 @@ resource "aws_instance" "app_server" {
       "sudo apt-get install -y mysql-client-8.0",
       "sudo apt-get install -y postgresql-client-11",
       "sudo apt-get install -y prometheus-node-exporter",
+      "sudo NEEDRESTART_MODE=a apt-get install -y redis-tools",
```

Comment:
What does `NEEDRESTART_MODE=a` mean?

Reply:
Haha this was a fun one. So if you try to install it manually, a screen comes up and it asks you what services you want to restart post installation. This `NEEDRESTART_MODE=a` is the way to automatically say ok to that prompt. `-y` does not take care of this case.